osu.Game/Overlays/Dashboard/Home/News/NewsPostDrawableDate.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Cursor;
+using osu.Game.Graphics;
+using osu.Framework.Graphics;
+
+namespace osu.Game.Overlays.Dashboard.Home.News
+{
+    public abstract class NewsPostDrawableDate : CompositeDrawable, IHasCustomTooltip
```

Comment:
This I definitely disagree with existing. This base class does nothing useful.

IMO this should be removed, inheritors should implement `IHasCustomTooltip` themselves. Not too many lines of overhead.